### PR TITLE
rn2252

### PR DESCRIPTION
**Pull request by levshamin**

### CODE REVIEW

File: content/_releases/22/52.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  - /releases/22.52.html
+---
+
+## Features
+
+### Additing tenant roles
+
+We have changed the name of permission `tenants.tenant.edit_roles` to `global.tenant.edit_roles`.
+
+Now `PATCH /v2/tenants/:tenantId/roles` allows to create a new role with permissions in a tenant scope. We have also updatet API endpoint `PATCH /v2/tenants/{tenant_id}/members/{user_id}` which allows to grant any role from tenants scope to any tenant user. This will allow to give any user `tenants.user.generate_one_time_token` permission.
```
Comment (andrewreshitko) [inline content/_releases/22/52.md:19]: We have also updated API endpoint `PATCH /v2/tenants/{tenant_id}/members/{user_id}` which allows granting any role from the tenant's scope to any user. With this endpoint the permission `tenants.user.generate_one_time_token` can be granted to any user inside the tenant.

File: content/_releases/22/52.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+### Abbility to read recipies by SERVICE_ACCOUNT
+
+We try to make working with recipes as convenient as possible for our users. In a recent release [22.48](/releases/22/48#initial-setup), we have already begun some improvements in this direction. Now we are introducing a new permission `GLOBAL.RECIPE.GET` to view **all** recipes based on permissions's scope and gave the new permission to `SERVICE_ACCOUNT` with Global level. This is the first step in the development of a new feature, in which the new role of the superuser will allow you to review and approve recipes, as well as increase their level up to tenant or global if necessary.
```
Comment (andrewreshitko) [inline content/_releases/22/52.md:23]: Our goal is to make easy-to-use recipes. In a recent release [22.48](/releases/22/48#initial-setup), we have already begun some improvements in this direction. Now we are introducing a new permission `GLOBAL.RECIPE.GET` to view **all** recipes based on the permission's scope and provide the new permission to `SERVICE_ACCOUNT` with the Global level. This is the first step in the development of a new feature, in which the new role of the superuser will allow you to review and approve recipes, as well as to change their level up to tenant or global if necessary.

File: content/_releases/22/52.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+> **Please Note:** All updates related to the API are detailed [here]({{site.data.tenant.apiDocsUri}}/v2).
+
+{% if site.data.tenant.name == "elastic.io" %}
+
+## OEM Related
+
+### Scalable Redis Cluster
+
+We are working on improving the Redis Cluster. We need to be able to scale Redis Cluster if the load increases. In connection with this we made it scalable. Please follow the instructions below to deploy the cluster:
```
Comment (andrewreshitko) [inline content/_releases/22/52.md:33]: We are working on the Redis Cluster improvements. We made Redis Cluster scalable according to the load increases. 
Please follow the instruction below to deploy the cluster:

File: content/_releases/22/52.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+New secrets config variable:
+
+`REDIS_CLUSTER_PASSWORD` - the password of redis cluster instances.
+
+{% endif %}
+
+## Fixed bugs
+
+*   **FIXED** bug, where additional fields can't be saved in faceless credentials.
+*   **FIXED** bug, where the step number was duplicated. You can change the name of the step, but it won't include the step number.
```
Comment (andrewreshitko) [inline content/_releases/22/52.md:49]: *   **FIXED** bug, where the step number was duplicated. Now you can change the name of the step, but the step number will remain displayed.

File: content/_releases/22/52.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+*   **FIXED** bug with red Router branch indicator. Now when you have unconfigured Router branch it's panel has a red border:
```
Comment (andrewreshitko) [inline content/_releases/22/52.md:50]: *   **FIXED** bug with red Router branch indicator. Now if you have unconfigured Router branch its panel has a red border:

**Review by andrewreshitko — APPROVED** [review]

**Review by HannaTrotsenko — APPROVED** [review]

**Pull request merged**